.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: Run Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron:  '0 0 * * *'
```

Comment:
is it necessary to run tests everyday if tests are not related with any other components/database/api etc? maybe it should be just run on commit and on PR?

Reply:
I agree, what do you think @vlussenburg? In the ticket it asks for daily test runs but it might not be necessary

Reply:
The rationale for daily unit tests is the possibility of them magically stopping to work. IE maybe a dependency is out of date, maybe there's a weird dependency on an external service, maybe a unit test fails every 5th of the month. It's (practically) free to run tests daily and it make sure that when an engineer needs to make a change (remember we don't touch our SDKs weekly or even monthly sometimes) that they are sure it's still working instead of having to spend hours fixing unit tests that stopped working.

Please keep it as such :)

Reply:
Will keep it as such, thank you @vlussenburg !